files/grest/rpc/assets/asset_txs.sql

```diff
@@ -1,7 +1,8 @@
-CREATE FUNCTION grest.asset_txs (
+CREATE OR REPLACE FUNCTION grest.asset_txs (
```

Comment:
Didn't we remove OR REPLACE everywhere as all are dropped anyway by setup-grest?

Reply:
True, some functions still have them - I think it prolly helps have them in especially when testing code as-is without running setup-grest, but happy to get it removed for consistency

Reply:
I'm happy to keep them in as it doesn't hurt to have them, and agree, helps with testing.